code/hud/hudconfig.h

```diff
@@ -66,11 +69,38 @@ typedef struct HUD_CONFIG_TYPE {
 
 extern HUD_CONFIG_TYPE HUD_config;
 
-void hud_config_init();
-void hud_config_do_frame(float frametime);
-void hud_config_close();
-
-void hud_set_default_hud_config(player *p);
+struct HC_gauge_region
+{
+	const char		*filename;
+	int			x,y;
+	int			hotspot;
+	int			use_iff;
+	int			can_popup;
+	int			bitmap;
+	int			nframes;
+	int			color;
+	UI_BUTTON	button;
+
+	HC_gauge_region(const char *name, int x1, int y1, int h, int iff, int cp, int b, int nf, int cl) : filename(name), x(x1), y(y1), hotspot(h), use_iff(iff), can_popup(cp), bitmap(b), nframes(nf), color(cl){}
+};
+
+extern struct HC_gauge_region HC_gauge_regions[GR_NUM_RESOLUTIONS][NUM_HUD_GAUGES];
```

Comment:
eh, yes, sorry about that.  Doesn't actually need to be doxied, but `extern`'s should have a comment or comment header stating where they are defined so we know where to look for them.

Reply:
In that case, done.

Reply:
Sorry about the back and forth, just woke up from a nap.

If these members are defined within the associated .cpp (such as hudconfig.cpp) then the `extern` declarations can be doxied the same as you would with function or class declarations, since they are effectively in the same global scope.